internal/server/data/access_key.go

```diff
@@ -94,10 +95,9 @@ func DeleteAccessKeys(db *gorm.DB, selectors ...SelectorFunc) error {
 		return err
 	}
 
-	ids := make([]uid.ID, 0)
-	for _, k := range toDelete {
-		ids = append(ids, k.ID)
-	}
+	ids := slice.Map[models.AccessKey, uid.ID](toDelete, func(k models.AccessKey) uid.ID {
+		return k.ID
+	})
```

Comment:
I wouldn't block merging on this, but I definitely feel the old for loop was easier to read than the generic `slice.Map`.  Do we really need this?